java-squid/src/main/java/org/sonar/java/model/statement/TryStatementTreeImpl.java

```diff
@@ -116,6 +125,21 @@ public TryStatementTreeImpl completeStandardTry(InternalSyntaxToken tryToken, Bl
     return this;
   }
 
+  public TryStatementTreeImpl completeTryWithResources(InternalSyntaxToken tryToken, InternalSyntaxToken openParenToken, ResourceListTreeImpl resources,
+    InternalSyntaxToken closeParenToken, BlockTreeImpl block, List<CatchTreeImpl> catches) {
+    this.tryToken = tryToken;
+    this.openParenToken = openParenToken;
+    this.resources = (List) resources;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unnecessary cast to "List". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1905)